replica/distributed_loader.cc

```diff
@@ -462,6 +462,14 @@ future<> table_populator::populate_subdir(sstables::sstable_state state, allow_o
     });
 }
 
+future<> distributed_loader::populate_keyspace(distributed<replica::database>& db, keyspace& ks, sstring ks_name) {
+    // might have more than one dir for a keyspace iff data_file_directories is > 1 and
+    // somehow someone placed sstables in more than one of them for a given ks. (import?)
+    return parallel_for_each(db.local().get_config().data_file_directories(), [&db, &ks, ks_name] (const sstring& data_dir) {
+        return populate_keyspace(db, ks, data_dir, ks_name);
+    });
+}
+
 future<> distributed_loader::populate_keyspace(distributed<replica::database>& db, keyspace& ks, sstring datadir, sstring ks_name) {
```

Comment:
minor: almost the same signature, third parameter has the same type, but different meaning. Not so important since it's more or less internal.

Reply:
Even worse -- the `datadir` parameter is erroneously unused (#15418)